README.md

```diff
@@ -1,7 +1,19 @@
-# AppBuilder SSG Template for Edge Delivery Storefront
+# Adobe Edge Delivery Services Storefront - PDP Optimizer (Backend Pack)
```

Comment:
This will go beyond PDP (some customers already adapted it for PLP, hreflang, etc)
proposed: Adobe Edge Delivery Services Storefront - pre-rendering dynamic content